doc/manual/parallel-computing.rst

```diff
@@ -860,7 +860,7 @@ If ``io`` is not specified, ``host`` and ``port`` are used to connect.
 For example, a cluster manager may launch a single worker per node, and use that to launch
 additional workers. ``count`` with an integer value ``n`` will launch a total of ``n`` workers,
 while a value of ``:auto`` will launch as many workers as cores on that machine.
-``exename`` is the name of the julia executable including the full path.
+``exename`` is the name of the Julia executable including the full path.
```

Comment:
I was thinking that "Julia" would refer the the language while "\`julia\`" refers to the command-line program. Does that seem sensible?